src/BullaClaim.sol

```diff
@@ -539,7 +519,7 @@ contract BullaClaim is ERC721, EIP712, Owned, BoringBatchable {
         // TODO: EIP-1271 smart contract signatures
         bytes32 digest;
         {
-            uint256 nonce = approvals[owner][operator].createClaim.nonce;
+            uint256 nonce = approvals[owner][operator].createClaim.nonce++; // spec.RES1
```

Comment:
does this mutate the nonce and return it to the var `nonce`? If so, is it more efficient?

Reply:
it returns the current nonce, then mutates it. doing `++DECLARATION` is slightly more efficient than `DECLARATION++`